Kitodo/src/main/java/org/kitodo/services/schema/SchemaService.java

```diff
@@ -0,0 +1,707 @@
+        if (language.equals("ru")) {
+            metaOrg.setValue(titleRu);
+            metaTrans.setValue(titleOther);
+        } else {
+            metaTrans.setValue(titleRu);
+            metaOrg.setValue(titleOther);
+        }
+
+        if (metaOrg.getValue() != null && metaOrg.getValue().length() > 0) {
+            inStruct.addMetadata(metaOrg);
+        }
+        if (metaTrans.getValue() != null && metaTrans.getValue().length() > 0) {
+            inStruct.addMetadata(metaTrans);
+        }
+    }
+
+    /**
+     * Remove all unnecessary persons.
+     *
+     * @param inStruct
+     *            DocStruct object
+     */
+    private void rusdmlDropPersons(DocStruct inStruct) {
+        if (inStruct.getAllPersons() != null) {
+            List<Person> copy = new ArrayList<>(inStruct.getAllPersons());
+            for (Person copyPerson : copy) {
+                if (copyPerson.getType().getName().equals("ZBLAuthor")) {
+                    inStruct.getAllPersons().remove(copyPerson);
+                }
+            }
+        }
+    }
+
+    /**
+     * Modify all metadata to modify.
+     *
+     * @param inStruct
+     *            DocStruct object
+     */
+    private void checkMetadata(DocStruct inStruct, Prefs prefs) {
+        if (inStruct.getType().getName().equals("Illustration")) {
+            DocStructType dst = prefs.getDocStrctTypeByName("Figure");
+            inStruct.setType(dst);
+        }
+    }
+
+    private void correctPathImageFiles(DocStruct phys, Prefs prefs, String newValue) throws ExportFileException {
+        MetadataType mdTypeForPath = prefs.getMetadataTypeByName("pathimagefiles");
+        List<? extends Metadata> allMetadata = phys.getAllMetadataByType(mdTypeForPath);
+        if (allMetadata.size() > 0) {
+            for (Metadata meta : allMetadata) {
+                meta.setValue(newValue);
+            }
+        } else {
+            throw new ExportFileException("Export error: Image path not yet set");
+        }
+    }
+
+    /**
+     * Maintain the central project settings in the xml configuration.
+     *
+     * @param topStruct
+     *            DocStruct object
+     * @param process
+     *            Process object
+     */
+    private void addMissingMetadata(DocStruct topStruct, Prefs prefs, Process process) throws ExportFileException {
+        String ppn = BeanHelper.determineWorkpieceProperty(process, "PPN digital");
+        if (ppn.length() == 0) {
+            throw new ExportFileException("Export error: No PPN digital present");
+        }
+        addMissingMetadata(topStruct, prefs, process, ppn);
+    }
+
+    /**
+     * Add missing metadata for RUSDML.
+     *
+     * @param inTopStruct
+     *            DocStruct object
+     * @param process
+     *            Process object
+     * @param ppn
+     *            String
+     */
+    private void addMissingMetadata(DocStruct inTopStruct, Prefs prefs, Process process, String ppn) {
```

Comment:
Parameter `prefs` is missing in Javadoc. Please add this parameter.

Reply:
done